[IMP] Inventory: Bringing least packages current

erjer-odoo opened the pull request:
While working on the removal strategies overview script, I realized that the documentation for "Least packages removal" was out of date. In this PR, I did the following:

- Added a section about enabling the removal strategy
- Added a subsection about storing products in packages before they can be used
- Added subsections on enabling the strategy on the location or product category
- Updated screenshots to be current
- Updated language to be current with 18.3 forward

All of this work makes the last of the removal strategies docs consistent with the other removal strategies docs.

This saas-18.3 PR can be FWP up to master.

erjer-odoo commented:
Hi @larm-odoo ! I'm finishing up work on the removal strategies docs. This update brings it current, as well as consistent with the other removal strategies docs that I worked on last week. When you get a moment, can you review and let me know what you think? Thank you!

larm-odoo commented on content/applications/inventory_and_mrp/inventory/shipping_receiving/removal_strategies/least_packages.rst:
```diff
@@ -5,10 +5,6 @@ Least packages removal
-   - :doc:`About removal strategies <../removal_strategies>`
-   - `Odoo Tutorials: Least Packages <https://www.odoo.com/slides/slide/5477/share>`_
-
 To understand how the removal strategy works, consider the following example, featuring a warehouse
```
I feel the whole example belongs in an admonition block, because currently it's "imagine if.." then "example: ..." but they are both the same example, so combining them might be visually easier to follow.

larm-odoo commented on content/applications/inventory_and_mrp/inventory/shipping_receiving/removal_strategies/least_packages.rst:
```diff
@@ -26,29 +22,82 @@ strategy is used to pick from a single, opened package.
+removal strategy.
+
+Navigate to :menuselection:`Inventory app --> Configuration --> Settings` and select the check box
+for the :guilabel:`Packages` feature.
```
```suggestion
Navigate to :menuselection:`Inventory app --> Configuration --> Settings` and in the :guilabel:`Operations` section, activate the check box
for the :guilabel:`Packages` feature.
```
I'll be 110% honest- I cannot remember if it was select or enable, etc. This is just a personal preference- both the extra navigation and 'select'. I realie it's the first thing, but INventory has a LOT of secitons so it might be helpful.

larm-odoo commented on content/applications/inventory_and_mrp/inventory/shipping_receiving/removal_strategies/least_packages.rst:
```diff
@@ -26,29 +22,82 @@ strategy is used to pick from a single, opened package.
+------------------------
+
+After you have configured your product inventory, set the removal strategy on the product category
+or storage location.
```
Sily nitpick- I woud swap the order of 'storage location' and 'pproduct form since they are listed below in the opposite order. Also, I recommend adding the ref's to the sections, and linking them so the reader can click the removal strategy method and read th eone they want. I go crazy with Ref's, so you can ignore that one! But the re-order I do think is important.

larm-odoo commented on content/applications/inventory_and_mrp/inventory/shipping_receiving/removal_strategies/least_packages.rst:
```diff
@@ -26,29 +22,82 @@ strategy is used to pick from a single, opened package.
+Configuration --> Categories` and selecting the intended product category. Next, in the
+:guilabel:`Force Removal Strategy` field, specify :guilabel:`Least Packages`.
+
+.. image:: least_packages/category-removal-strategy.png
```
I recommend completing an image whenever possible. I know you don't NEED a parent category, tax category, or route- but if you can add them, and it makes sens,e I always feel like it strengthens the example. So for this one, your 'Baking Materials coul dhave a parent category of "dry goods" or whatever woudl make sense 9I don't think my example is perfect, so think about it!) but I just think it's a nice touch to fully flesh-out the images with a real-world scenario.

larm-odoo commented on content/applications/inventory_and_mrp/inventory/shipping_receiving/removal_strategies/least_packages.rst:
```diff
@@ -26,29 +22,82 @@ strategy is used to pick from a single, opened package.
 .. important::
-   The :ref:`Packages feature <inventory/warehouses_storage/pack-setup>` **must** be enabled to use
-   this strategy.
+   When there are different removal strategies applied on both the location and product category for
```
Very good to know! You could expand this with an example, and say the location jas this rmeoval, the product has that removal, and then say which is used. I am trying to think of a good reason they would be different and it woud make sense (maybe... the category coul dbe for something expired but the locaiton is set to something else for some reason) I am trying to think WHY somoene wold have them be different- I know it exists- I just can't think of it myself right now =)

larm-odoo commented on content/applications/inventory_and_mrp/inventory/shipping_receiving/removal_strategies/least_packages.rst:
```diff
@@ -58,15 +107,18 @@ the delivery order is created.
+In the :guilabel:`Detailed Operations` pop-up window, the :guilabel:`Pick From` field displays where
+the quantities to fulfill the :guilabel:`Demand` are picked. Because the order demanded eighty
 kilograms, which exceeds the quantity in the opened package of `54 kg`, an unopened package of `100
 kg` is selected.
```
Basically, is there another way to veirfy/see what bag was used for this order? Maybe it's on another screen or report? I'm just thinking out loud, because if I was a worker, from this image, I wouldn't know that a 100kg bag was used- I could think that a 50 and a 50 were used/combined.

erjer-odoo commented on content/applications/inventory_and_mrp/inventory/shipping_receiving/removal_strategies/least_packages.rst:
```diff
@@ -26,29 +22,82 @@ strategy is used to pick from a single, opened package.
+   :alt: Enable Storage Locations and Multi-Step Routes to set removal strategies on locations.
+
+Next, open :menuselection:`Inventory app --> Configuration --> Locations`. Open an existing location
+or create a new location. On the location form, under the :guilabel:`Logistics` heading, set
```
Unfortunately, that heading doesn't have a ref link to it, so I can save this for a future 1-pt update. I feel like it's fairly straightforward -- they just click the "New" button, and then the relevant fields in the form are described here.

larm-odoo replied:
Totally understandable! In the future, if you DO want to link - yo ucan add a ref tag to wherever in another doc you want, and point ot it. =) But I agree- it's very straightforward, it was justa suggestion. But odn't let the absence of an anchor prevent you form adding a ref in the future- you can just add them to the needed docs =)

larm-odoo commented on content/applications/inventory_and_mrp/inventory/shipping_receiving/removal_strategies/least_packages.rst:
```diff
@@ -26,47 +23,109 @@ strategy is used to pick from a single, opened package.
+
+After you have configured your product inventory, set the removal strategy on the :ref:`storage
+location <inventory/warehouses_storage/lp-storage-location>` or :ref:`product category
+<inventory/warehouses_storage/lp-product-category>`.
```
1- I LOVE these! It makes me happy, I love clicking and jumping ot toher places in docs!
2- the Ref's seem long - if you want, you can shorten them. They don't have to have the full fil epath - they just need to be unique, meaning no other doc can have the same exact anchor tag. You coudl cshorten these to "inventory/lp-location" for example. If you accidentally use a tag/ref anchor that is already in the databs,e the terminal will catch it during 'make rview' - and then you can change them. That's happened to me when I use generic "payroll/rules" or osmethign that may come up in another doc. Just a suggestion!

erjer-odoo commented:
Hi @Felicious -- I've addressed the concerns raised by @larm-odoo, and she's approved what's there. When you have a moment, could you take a look? If you also approve, could you help me merge? Thank you!

Felicious commented:
Good work here, @erjer-odoo !

@robodoo r+
